### DISCUSSION TITLE
2024-07-26 - ✨ SysAdmin Appreciation Day 2024 - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by bradystroud** [SSWConsulting/SSW.Rules.Content#8934] [Chewing the Fat]
Happy SysAdmin Day!
https://sysadminday.com/

https://youtu.be/07Ed0qUcxMs
https://youtu.be/fwTLhC8K5Uw

What is an awesome (or bad) experience you had dealing with a SysAdmin?

**Comment by Freego1783** [comment]
In my previous job only one of the SysAdminss was good and when he was not around nothing was being done or fixed. It led to critical situations.

On the other hand, I had only good experiences with the SysAdmins at SSW. Thank you guys for your hard work 😁

**Comment by JeanThirion** [comment]
Funny enough I didn't really have any SysAdmins in my last 3-4 Consulting companies - we basically sorted things out ourselves.

I've had great SysAdmins at clients' though, helping with deployments hell, or fixing stuff for me.

**Comment by tiagov8** [comment]
Raising an SSW SysAdmin Zendesk ticket means the problem will be solved.

**Comment by Levijj22** [comment]
Awesome team - thanks guys!
#FIXITKIKI

![IT Crowd](https://github.com/user-attachments/assets/870bd214-ca08-4db9-9ee9-a5b993e5c4d5)

**Reply by kikibianc** [reply]
![1520225634705](https://github.com/user-attachments/assets/eb44b263-d420-43d1-9bd8-13f83d0e3e49)

**Comment by jeoffreyfischer** [comment]
Of course Sys Admin are essential, but how can we talk about all that computer stuff, all that computer gobbledygook, without saying a word about...

![784cb237-e558-4ec2-9cb4-2376e3f3387e_text](https://github.com/user-attachments/assets/f14e044e-e662-4272-9d95-091b85954fad)

![c34eb81d-0850-4cd4-a7b3-78990b913b77_text](https://github.com/user-attachments/assets/6e15a8da-264c-4998-85fc-1c3eb2e664d8)

**Comment by leahy268** [comment]
I've been a Sysadmin now for 20 years and I've seen a lot of good and bad ones.  I could give some war stories from the basement :).

**Comment by jakebayliss** [comment]
I miss ash

**Comment by uly1** [comment]
![image](https://github.com/user-attachments/assets/7309d5a3-2a4e-4bf0-98eb-5c385c628e15)

Fix it fix it fix it fix it.... fix it!

Thanks lads!

**Comment by MarcusIrmscherSSW** [comment]
It's always a pleasure dealing with the SSW SysAdmins 🔧

**Comment by ricksu978** [comment]
On the Microsoft Outage day, the MyAccess Package page wasn't loading and I was blocked to work on YakShaver project.

I created Zendesk ticket ([#14548](https://pdi-ssw.zendesk.com/hc/en-us/requests/14548)) and Kiki immediately add me to the requested Security Group so that I can start my day!

**Comment by Ben0189** [comment]
SSW sysadmin is awesome, they resolved all the issues I had raised and quick response

**Comment by steven0x51** [comment]
Thank you Sysadmin team, Awesome team!
![A-person-holding-a-phone-to-his-ear-300x168](https://github.com/user-attachments/assets/09607e07-86cc-47ce-a317-76eaa15cb882)

**Comment by TobyChurches** [comment]
Our SysAdmins are amazing! Always super friendly and quick to help (Even when I don't know what I'm supposed to be asking 😅)

**Comment by Tanya-Leahy** [comment]
Thank you, SSW SysAdmin team. You guys are amazing! My tickets are always completed in record time.

**Comment by amankumarrr** [comment]
It's a challenging job, but our SysAdmins are working diligently to resolve the issues.

**Comment by matt-goldman-ssw** [comment]
Thanks to our sysadmins, I know that if you type 'google' into Google you will break the internet (which fortunately does not weigh anything)

**Comment by Calinator444** [comment]
> What is an awesome (or bad) experience you had dealing with a SysAdmin?

This one time I requested access to a power automate flow on a 10 minute call with @amankumarrr, before the call is over @chrisschultzssw had already approved my access package request.

**Comment by tkapa** [comment]
Our SysAdmins always seem to be on top of their ZenDesk requests, and will occasionally make great recommendations when something can be improved easily.

**Comment by kikibianc** [comment]
Thank you for all the love, SSW! 
SysAdmins are definitely on a crusade to get all tickets, old and new, prioritized and solved in record time, with the best possible experience for you 😎

![images](https://github.com/user-attachments/assets/6d5bdbe3-5d85-4eb4-a98b-080b605d1ba4)

... even when we're not very sure ourselves 😁

**Comment by chrisschultzssw** [comment]
💖

![image](https://github.com/user-attachments/assets/fbf15b3b-8647-439a-8fdf-7ce8a3cb55a7)

**Comment by GordonBeeming** [comment]
I've normally mostly had good experiences with SysAdmins throughout all my roles.
Our SysAdmins are great, they have a lot of services to look after between a couple of them

**Comment by awaltos** [comment]
Thank you @kikibianc @chrisschultzssw @leahy268 @ozdemir-mehmet ! I have always had pleasant experiences dealing with SSW Sys Admins ❤️ 

Sys Admins often have solid cloud knowledge and can help not only in keeping systems running but also in advising on solutions and best practices for new/existing projects.

**Comment by PennyWalker** [comment]
Thank you guys! I've always had the best experience working with you, and you've always gone out of your way to help me even if you've been busy dealing with other things. It doesn't go unnoticed, and I'm really grateful!

I've seen you guys work through some serious stuff and come out the other end with a fantastic attitude and lots of learnings. Thank you for diligently working through 75 recommendations to make us more secure - I know how much that took out of you! <3

**Comment by Dhruv-0987** [comment]
What is an awesome (or bad) experience you had dealing with a SysAdmin? 
All experiences with SysAdmins are good, specially with Chris S, he is the best always so calm and chill, he helped me made my mac compliant with our new security standards even though the instructions were right there haha, thanks @chrisschultzssw 😄

**Comment by JackDevAU** [comment]
![HVe](https://github.com/user-attachments/assets/9a24928b-0f70-4b15-a5d6-f1f7e318bc0e)
**Figure: Normal SysAdmin waiting times**

![image](https://github.com/user-attachments/assets/38500c5f-33ab-41af-a757-d7e6e222ea4f)
**Figure: Me never having to wait for SSW SysAdmins**

**Comment by MattParkerDev** [comment]
SysAdmins keep the world turning! 🌍

**Comment by christoment** [comment]
I have never had bad experience with SysAdmins - all experiences are good experience 😄

Cheers to our SysAdmins guarding our infrastructure! 🍻

**Comment by bettybondoc** [comment]
At my old job, everything strictly had to go through a ticketing system (via a portal), which was fine until your laptop dies and you can't file a ticket 🫠 (good luck reaching the helpline)

It blew my mind when I first encountered SSW's rule: [call your SysAdmin first](https://www.ssw.com.au/rules/call-your-system-administrators-before-raising-a-ticket/). Many issues were resolved in minutes with a quick chat or call. No more endless ticketing back-and-forth.

**Comment by AntPolkanov** [comment]
Always a pleasure talking to our SysAdmins.
You are the best, guys! 💪

![image](https://github.com/user-attachments/assets/ee8a91c0-abf6-47bc-8178-672ae0e14ac1)

**Comment by duanxinhuan** [comment]
I really want to say thank you to Chris and Ash.
They are so supportive and nice to me on YakShaver!
I know I'm anoyying and call them all the time, but there is not even once they are not patient and supportive!

**Comment by sethdaily** [comment]
✅ Watching Mehmet shoot hoops at lunch 🏀 after decimating the chest press machine for 4 reps of 120kg

**Comment by will-truscott-ssw** [comment]
Had an issue signing into the wifi on the Oculus Quest solved within 10 minutes of raising it. Brilliant team!

**Comment by jerryluossw** [comment]
Always have great experience with SysAdmins, thank you SSW Sysadmin team 😀😀

**Comment by stephanfakossw** [comment]
Happy SAD 😊

![FOreverOnlineGIF](https://github.com/user-attachments/assets/974f94bf-aaf5-4074-a61a-7b449f431d4e)

**Comment by Marxoz** [comment]
Happy SAD legends!

**Comment by jimmidier** [comment]
Happy SAD SysAdmins and happy for you to have everyone's love!

**Comment by rubycogan** [comment]
I love seeing SysAdmins fix a problem using Google (I'd already googled it myself but was unsuccessful).

**Comment by Aibono1225** [comment]
Happy SAD! Thanks for everything ❤️

**Comment by miichaelsmedley** [comment]
![1356bee27fb30f1416df2f4c433a23378fd1cdf9](https://github.com/user-attachments/assets/4d295eb8-9fca-435b-a3b4-b83baab3a3ca)

**Comment by Hona** [comment]
SysAdmins are important now more than ever!

**Comment by KristenHu** [comment]
Happy SAD! Thanks for all the things you guys did!
![image](https://github.com/user-attachments/assets/e8bcc6b9-55a5-425b-a960-45bfde9b4288)

**Comment by babakamyljanovssw** [comment]
I've had only good experience with SysAdmins, thanks for your effort to keep services up and running! Happy SAD!

**Comment by bahjatalaadelssw** [comment]
I needed to test and label ports in the patch panels in the Sydney Chapel. On the same day Kiki and Ash came up with tools to trace and label so I could save on cabling and use existing connections. And one of these days Chris S will get the tools he needs to get the Sydney TV's behaving themselves.

**Comment by PothieuG** [comment]
Happy SysAdmin Day !
No bad experience at SSW, thanks for everything !

**Comment by vladislav-kir** [comment]
I still cannot believe how great SSW SysAdmins are! Always so helpful!!! Thank you guys! 🤩

**Comment by danielmackay** [comment]
At a previous company we didn't have a local SysAdmin team, we did have an overseas one.  They once decided to enable endpoint protection on my machine without telling me.  However, it wasn't setup properly and bricked my laptop for 1.5 days during which time I couldn't do any work all.  Worse yet, there wasn't even anyone I could call as there were all asleep.  Worst.  Experience. Ever! 😤

**Comment by drwharris** [comment]
Nothing but great experiences with Sys Admins at SSW

**Comment by ozair-ashfaque** [comment]
Happy belated SAD

**Comment by SamBWagner** [comment]
I think the SSW Sysadmins are super awesome! Good lil' team of good lil' lads.
